### PR TITLE
added toString() before toLowerCase()

### PR DESCRIPTION
**Pull request by ctnkaan**
I did this because Discord API gives an error if you try to do toLowerCase() before not doing toString(). This PR allows this project to be used with Discord bots.
Tested it on my personal Discord Bot. It works without any issues.

Hope it helps. Thank you for this awesome project !

### CODE REVIEW

**Comment by ctnkaan** [comment]
image

Here is the results before the commit and after the commit

**Review by plainheart — COMMENTED** [review]
Thanks! I didn't consider the case in which the input parameter is not a string. It may be slightly better if we also judge if the `lang` parameter is in string type.



https://github.com/plainheart/bing-translate-api/blob/c1d080f56bdeb239fa3f8d9dce4e8325c09bc240/src/lang.js#L123

**Comment by ctnkaan** [comment]
image

Hello again. I hope I understand what you mean correctly. Do you mean something like this ? If so I can open a new PR with this.
I tested it and it seems to work without any issues. Hope I got it right.

File: src/lang.js [hunk 1]
```diff
@@ -120,15 +120,15 @@ const LANGS_CAN_CORRECT = [
 ]
 
 function getLangCode(lang) {
-  if (!lang) {
+  if (!lang.toString()) {
```
Comment (plainheart) [inline src/lang.js:123]: The condition `!lang` is still required and we need to add another condition to judge if it is a string.
```js
!lang || typeof lang !== 'string'
```

File: src/lang.js [hunk 2]
```diff
@@ -120,15 +120,15 @@ const LANGS_CAN_CORRECT = [
...
     return
   }
 
   if (LANGS[lang]) {
     return lang
   }
 
-  lang = lang.toLowerCase()
+  lang = lang.toString().toLowerCase()
```
Comment (plainheart) [inline src/lang.js:131]: This change is not needed any longer.

**Review by plainheart — COMMENTED** [review]
Sorry, I didn't make it clear. I mean that we need to add an additional condition to judge if `lang` is in string type. See the following review comments.

**Comment by ctnkaan** [comment]
Ok I think I made the changes you mentioned and pushed them to this PR. Discord API still does not give an error.
Sorry for the late response. It was like 6 AM at my local time so I went to bed. Hope I did it right. Thank your your great explanation at the comments.
image

**Review by plainheart — APPROVED** [review]

**Comment by plainheart** [comment]
Thanks a lot for your work!

**Pull request merged**